CHANGELOG.md

```diff
@@ -212,6 +212,7 @@ Released January 31, 2019
 - Serialization of States will _ignore_ any result data that hasn't been processed - [#581](https://github.com/PrefectHQ/prefect/pull/581)
 - Removes `VersionedSchema` in favor of implicit versioning: serializers will ignore unknown fields and the `create_object` method is responsible for recreating missing ones - [#583](https://github.com/PrefectHQ/prefect/pull/583)
 - Convert and rename `CachedState` to a successful state named `Cached`, and also remove the superfluous `cached_result` attribute - [#586](https://github.com/PrefectHQ/prefect/issues/586)
+- Removes `prefect make user config` from cli commands [904](https://github.com/PrefectHQ/prefect/issues/904)
```

Comment:
I think you accidentally put this in the wrong "Breaking Changes" section; there is one section per release, so this one should go in the top "Unreleased" section

Also, this is super minor but add a hypen after `commands` and a pound sign before the number so it's like 
```
from cli commands - [#904](https://github.com/PrefectHQ/prefect/issues/904)
```